source/material_model/rheology/strain_dependent.cc

```diff
@@ -415,7 +477,9 @@ namespace aspect
         if  (this->simulator_is_past_initialization() && this->get_timestep_number() > 0 && in.requests_property(MaterialProperties::reaction_terms))
           {
             const double edot_ii = std::max(sqrt(std::fabs(second_invariant(deviator(in.strain_rate[i])))),min_strain_rate);
-            const double e_ii = edot_ii*this->get_timestep();
+            double e_ii = edot_ii*this->get_timestep();
+            if (healing_mechanism != no_healing)
```

Comment:
Can you add a short comment above this line describing what is happening? (i.e., adjusting the strain values to account for strain healing).